### ISSUE TITLE
The hyperparameters of the models

### ISSUE THREAD

**Issue by Youngkl0726** [Cadene/pretrained-models.pytorch#69]
Can you supply the hyperparameters you used to train the models, such as the learning rate, the batch size and the number of gpus you used?

**Comment by Cadene** [comment]
@Youngkl0726 We did not train any models. We ported/converted them from their original repo instead. It is usually difficult and sometimes impossible to reproduce the results obtained by one or another architecture. That is why we created this repo.
